### ISSUE TITLE
[FEATURE REQUEST] Use degassing-only volcano climatology for GEOS-Chem benchmarks

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#1005]
@christophkeller wrote:
>Hi Bob, Daniel, et al., 
>as mentioned by Daniel, it occurred to me that it would probably be better to use the climatological volcano emissions  degassing only) for future benchmarks as this is more representative for an average simulation - and also makes it easier to compare should the benchmark year change. I attach the latest climatology, and you can also find it here:
https://gmao.gsfc.nasa.gov/gmaoftp/geoscf/volcano_so2/so2_volcanic_emissions_CARN_v202005.degassing_only.rc
> 
>To run HEMCO with the climatology, one would simply point to the degassing file in HEMCO_Config.rc:
>
```
# degassing and eruptive:
#    --> Volcano_Table    :       $ROOT/VOLCANO/v2021-09/$YYYY/$MM/so2_volcanic_emissions_Carns.$YYYY$MM$DD.rc
# degassing only:
    --> Volcano_Table   :       $ROOT/VOLCANO/v2021-09/so2_volcanic_emissions_CARN_v202005.degassing_only.rc
```

**Comment by msulprizio** [comment]
I would recommend against hardcoding the `Volcano_Table` path in HEMCO_Config.rc to the degassing file as this will use the climatology emissions for all years/simulations and users might not be aware of this. Instead, I suggest changing the available end year for these emissions to 2018 and allowing the volcano extension to use the climatology file if outside that range. 

Another alternative would be to only replace the `Volcano_Table` path for benchmark simulations. This can be done using the automated run directory updates in PR https://github.com/geoschem/geos-chem/pull/459 (slated for 13.4.0 also). I would suggest creating a new `RUNDIR_` variable (e.g. `RUNDIR_VOLC_TABLE`) and changing the line in HEMCO_Config.rc.template.fullchem to:

```
    --> Volcano_Table   :       ${RUNDIR_VOLC_TABLE}
```

In the .txt settings files or in `createRunDir.sh` the path for `RUNDIR_VOLC_TABLE` would use the degassing-only file for benchmark simulations or the degassing+eruptive file for the current date otherwise.

**Comment by yantosca** [comment]
Thanks @msulprizio.  Maybe the easiest thing to do is to wait until the rundir updates get merged in, and apply the fix then.

**Comment by yantosca** [comment]
PR #1053 has now been merged into the 13.4.0 development stream.  We can now close this issue.
